lib/tweet/tweet.dart

```diff
@@ -298,17 +297,23 @@ class TweetTileState extends State<TweetTile> with SingleTickerProviderStateMixi
     });
   }
 
-  static const List<String> _twitterUrls = ['twitter.com', 'pic.twitter.com', 'twimg.com', 'abs.twimg.com', 'pbs.twimg.com', 'video.twimg.com'];
+  static const List<String> _twitterUrls = [
+    'twitter.com',
```

Comment:
I saw today a x.com short link on a twitter post

Reply:
I've added x.com to the list of urls, anything else needing to be changed?